Utils/SpoilerUtils.cs

```diff
@@ -59,17 +59,16 @@ public static void CreateSpoilerLog(RandomizedResult randomized, SettingsObject
 
             if (settings.GenerateHTMLLog)
             {
-                filename += "_SpoilerLog.html";
-                using (StreamWriter newlog = new StreamWriter(Path.Combine(directory, filename)))
+                using (StreamWriter newlog = new StreamWriter(Path.Combine(directory, filename + "_Tracker.html")))
```

Comment:
If the HTML spoiler log is going to be considered a tracker, do you think we should remove the "hover to reveal" aspect of it.

Reply:
That might be a good idea. I could go either way though.